src/Runner.Listener/Configuration/NativeWindowsServiceHelper.cs

```diff
@@ -1,4 +1,5 @@
 #if OS_WINDOWS
```

Comment:
dotnet 6 introduced api platform check at compile time.
We already did our platform check using `#if OS_WINDOWS`, so we can just disable the working everywhere we have `#if OS_WINDOWS`